src/html/parser.ts

```diff
@@ -639,8 +690,26 @@ export class Parser {
      */
     protected Text(token: Text): void {
         debug("[html] Text %j", token)
-
         const parent = this.currentNode
+        if (
+            !this.isUsingCustomTokenizer &&
+            token.value &&
+            parent.type === "VElement" &&
+            parent.name === "template"
```

Comment:
I think you can probably use `parent.parent.type` without using `isUsingCustomTokenizer`.

```suggestion
            token.value &&
            parent.type === "VElement" &&
            parent.name === "template" &&
            parent.parent.type === "VDocumentFragment"
```